### PR TITLE
Adding Works and About Myself Section

### PR DESCRIPTION
**Pull request by orozCoding**
Created the "My Works" and "About Myself" sections.

Edited the HTML file and also the CSS file in order to do so.

Still focused just on the mobile version.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -96,6 +97,226 @@ ul {
   justify-content: start;
 }
 
+/* My Works Section */
+
+.my-works {
```
Comment (keneogu) [inline css/styles.css:102]: - [ ] The my-works section should have margins as defined in the Figma design. I suggest you add margins to the styling.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @orozCoding, 

Good job so far!

- Linters working properly
- Well documented PR description
- Commit messages are meaningful
- Good HTML structure

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Your background image or your image doesn't display please fix it:
Your design
oroz
Figma design
oroz1


- [ ] Some of your icons don't display too. Please fix it.
Your design
oro
Figma design
oro1



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉✔**
Hello @orozCoding 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by orozCoding** [comment]
Thank you!

**Pull request merged**